public/app/features/datasources/components/picker/DataSourceList.tsx

```diff
@@ -54,7 +54,6 @@ export function DataSourceList(props: DataSourceListProps) {
   const styles = getStyles(theme, selectedItemCssSelector);
 
   const { className, current, onChange, enableKeyboardNavigation, onClickEmptyStateCTA } = props;
-  // QUESTION: Should we use data from the Redux store as admin DS view does?
```

Comment:
No, we should not. That endpoint returns the details of the Ds, not the DS settings.